### ISSUE TITLE
Nested tag and Markdown output

### ISSUE THREAD

**Issue by kohki-shikata** [seangravener/hexo-tag-htmlTag#2]
Hi.

I hope to use any html tags in Hexo markdown file, then I found this plugin.
It seems not work with block level, is my understanding correct?
Can not this plugin work for be nested?

input:
---
```
{% htmlTag div class="example" %}
Bra, bra, bra...

* Loren
* ipsum
{% endhtmlTag %}
```
---

However code above didn't work. An error occured.

 I expect below...

output:
---
```
<div class="example">
  <p>Bra, bra, bra...</p>
  <ul>
    <li>Loren</li>
    <li>ipsum</li>
  </ul>
</div>
```
---

What should I do to solve this problem?
Any ideas?

**Comment by seangravener** [comment]
It appears you're right. As a feature, [Markdown doesn't honor its syntax nested within block level html tags](https://daringfireball.net/projects/markdown/syntax#html).

> Note that Markdown formatting syntax is not processed within block-level HTML tags. E.g., you can’t use Markdown-style emphasis inside an HTML block.

I think for this tag to work with nested markdown, the markdown embedded within the tag will need to be rendered prior to inserting the block level parent tags. It might be possible to hook into to a Hexo renderer event to do this.

**Comment by seangravener** [comment]
Hey @kohki-shikata, looks like you have a fix in your fork. Feel free to submit a PR.

**Comment by seangravener** [comment]
Fixed in 9003777
